### ISSUE TITLE
Link to the github repositor(ies) from the site

### ISSUE THREAD

**Issue by rgbkrk** [IRkernel/irkernel.github.io#39]
The site should link back to https://github.com/irkernel _somewhere_.

**Comment by flying-sheep** [comment]
true. currently those links are in the sidebar, which is visible on small screens only.

where do you think they would fit?
